core/controlplane/config/templates/cluster.yaml

```diff
@@ -239,7 +239,7 @@ worker:
 #      # If omitted, public subnets are created by kube-aws and used for worker nodes
 #      subnets:
 #      - # References subnets defined under the top-level `subnets` key by their names
```

Comment:
@sathiyas So perhaps your assumption is that one could forget to uncomment:

```yaml
#      - # References subnets defined under the top-level `subnets` key by their names
```

in

```
#      subnets:
#      - # References subnets defined under the top-level `subnets` key by their names
#        name: ManagedPublicSubnet1
```

which results in an invalid yaml like:

```yaml
      subnets:
#      - # References subnets defined under the top-level `subnets` key by their names
        name: ManagedPublicSubnet1
```

Am I following you correctly?